src/scripts/process_au.py

```
@@ -0,0 +1,146 @@
+import json
+from datasets import load_dataset
+import re
+
+def clean_text(text):
+    if not text:
+        return ""
+    text = re.sub(r'\s+', ' ', text).strip()
+    return text
```

Ruff (0.15.6)

[warning] Unnecessary assignment to `text` before `return` statement

Remove unnecessary assignment

(RET504)